### ISSUE TITLE
Node identifiers should start at `0` (not `1`)

### ISSUE THREAD

**Issue by vruusmann** [jpmml/jpmml-sklearn#73]
Users may wish to reproduce Scikit-Learn node identifiers (as they are reported by the `DecisionTree(Classifier|Regressor).apply(X)` function).

**Comment by vruusmann** [comment]
Better yet, there should be an option to re-label nodes based on a user-supplied `dict` - dictionary keys would be Scikit-Learn node identifiers, and dictionary values would be the desired textual node identifiers.
